package.json

```diff
@@ -11,10 +11,9 @@
   ],
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",   
+    "lint": "eslint .",
     "test": "mocha --timeout 30000 test/*test.js",
     "posttest": "npm run lint"
-    
```

Comment:
That was update by `npm install strong-globalize --save`. And doesn't need to be reverted back.